### PR TITLE
Fix topk metrics

### PR DESCRIPTION
**Pull request by sergunya17**
### Pull Request FAQ
- [documentation](https://catalyst-team.github.io/catalyst/)
- [contribution guide](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md)
- [minimal examples section](https://github.com/catalyst-team/catalyst#minimal-examples)
- [changelog](https://github.com/catalyst-team/catalyst/blob/master/CHANGELOG.md) for main framework updates
- [Catalyst slack (#__questions channel)](https://join.slack.com/t/catalyst-team-core/shared_invite/zt-d9miirnn-z86oKDzFMKlMG4fgFdZafw) for issue discussion

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### Checklist
- [x] Have you updated tests for the new functionality?
- [x] Have you added your new classes/functions to the docs?
- [ ] Have you updated the [CHANGELOG](https://github.com/catalyst-team/catalyst/blob/master/CHANGELOG.md)?
- [x] Have you run [colab minimal CI/CD](https://colab.research.google.com/github/catalyst-team/catalyst/blob/master/examples/notebooks/colab_ci_cd.ipynb) with `latest` and `minimal` requirements?
- [x] Have you checked XLA integration with [single](https://colab.research.google.com/github/catalyst-team/catalyst/blob/master/examples/notebooks/Catalyst_XLA_single_process.ipynb) and [multiple](https://colab.research.google.com/github/catalyst-team/catalyst/blob/master/examples/notebooks/Catalyst_XLA_multi_process.ipynb) processes?

### CODE REVIEW

**Review by Scitator — COMMENTED** [review]
could you please add docs on metric logic under `self.batch/epoch metrics`? For example, when you specify AccuracyCallback, it would add `accuracy01` to the metrics storages

File: catalyst/metrics/_topk_metric.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+from typing import Any, Callable, Dict, Iterable, List
+
+import torch
+
+from catalyst.metrics._additive import AdditiveMetric
+from catalyst.metrics._metric import ICallbackBatchMetric
+
+
+class TopKMetric(ICallbackBatchMetric):
```
Comment (Scitator) [inline catalyst/metrics/_topk_metric.py:9]: could you please add it to the `catalyst/docs/api/metrics.rst`?

**Review by Scitator — DISMISSED** [review]

**Comment by Scitator** [comment]
🤔  we also need to add a new metric to `catalyst/metrics/__init__.py`
